returnn/tf/layers/basic.py

```diff
@@ -551,19 +551,20 @@ def __init__(self, activation, **kwargs):
       self.output.placeholder = self.output_before_activation.y
 
   @classmethod
-  def get_out_data_from_opts(cls, activation, **kwargs):
+  def get_out_data_from_opts(cls, sources, name, activation, **kwargs):
     """
+    :param list[LayerBase] sources:
+    :param str name:
     :param str activation:
     :rtype: Data
     """
-    # Get dtype based on inputs
-    out = super(ActivationLayer, cls).get_out_data_from_opts(**kwargs)
-    # Modify if needed based on activation function
+    # get dtype based on the inputs
+    out = get_concat_sources_data_template(sources, name="%s_output" % name)
+    # modify if needed based on activation function
     if activation == "abs" and out.dtype == "complex64":
       out.dtype = "float32"
     return out
 
-
```

Comment:
Why was this empty line removed? It is needed here.

Reply:
I dont know, I confused myself.